tests/5.0/atomic/test_atomic_acquire_release.F90

```diff
@@ -0,0 +1,56 @@
+!//===------ test_atomic_acquire_release.F90 --------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! Adapted from OpenMP examples acquire_release.2.c
+! When the atomic read operation on thread 1 reads a non-zero value from y,
+! this results in a release/acquire synchronization that in turn implies that 
+! the assignment to x on thread 0 happens before the read of x on thread 1. 
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1000
+
+PROGRAM test_atomic_acquire_release_program
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_VERBOSE(test_atomic_acquire_release() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION test_atomic_acquire_release()
+    INTEGER:: x, y, thrd, tmp, errors
+
+    x = 0
+    y = 0
+    errors = 0
+
+    OMPVV_INFOMSG("test_atomic_acquire_release")
+
+    !$omp parallel num_threads(2) private(thrd, tmp)
```

Comment:
Should we be checking if we got 2 threads ? What happens of the implementation gives us only 1 thread ?

Reply:
If there is only 1 thread, the test becomes a trivial case, where atomic constructs are not needed at all.